gui/src/streamwindow.cpp

```diff
@@ -96,6 +96,14 @@ void StreamWindow::mouseReleaseEvent(QMouseEvent *event)
 		session->HandleMouseEvent(event);
 }
 
+void StreamWindow::mouseDoubleClickEvent(QMouseEvent *event)
+{
+	ToggleFullscreen();
+
+	if(session)
+		session->HandleMouseEvent(event);
```

Comment:
Better just call `QMainWindow::mouseDoubleClickEvent(event)` instead of touching the session.

Reply:
Done!